### ISSUE TITLE
Hello. Status?

### ISSUE THREAD

**Issue by adamwaite** [foulkesjohn/BetSwift#9]
Hello.

I'm interested in writing some automated Betfair trading in Swift. I started a similar project and stumbled on this repo.

I just wanted to say hello and ask where you got to with this project, and what made you stop? 

I started writing code for the authentication part and hit the immediate stumbling block of their tricky certificate based auth. I noticed in the master branch you used Kitura, and dropped it in the development branch?

Thanks!
Adam

**Comment by foulkesjohn** [comment]
Sorry for the delay on the reply. I gave up as I moved onto some other unrelated side projects. I did have some success with this placing orders and seeing some good returns on it. I’d still give the usual warning of make sure you test anything using this library extensively before placing live orders though.

In the development branch my plan was to move over to NIO libraries instead of Kitura. I can’t 100% remember where I got up to but I’ll try and answer any questions you’ve got.
